docs/static/azure-module.asciidoc

```diff
@@ -0,0 +1,591 @@
+
+[cols="<,<,<",options="header",]
+|=======================================================================
+|Name |Description|Notes
+
+|azure.subscription 	|Azure subscription from which this data originates. 	|Some Activity Log events may not be associated with a subscription.
+|azure.group 	|Primary type of data. |Current values are either 'activity_log' or 'sql_diagnostics' 	
+|azure.category* 	|Secondary type of data specific to group from which the data originated 	|
+|azure.provider 	|Azure provider 	| 
+|azure.resource_group 	|Azure resource group 	| 
+|azure.resource_type 	|Azure resource type 	| 
+|azure.resource_name 	|Azure resource name 	| 
+|azure.database 	|Azure database name, for display purposes 	|SQL Diagnostics only
+|azure.db_unique_id 	|Azure database name that is garunteed to be unique 	|SQL Diagnostics only
+|azure.server 	|Azure server for the database 	|SQL Diagnostics only
+|azure.server_and_database 	|Azure server and database combined 	|SQL Diagnostics only
+|azure.metadata 	|Any @metadata added by the plugins, for example var.input.azure_event_hubs.decorate_events: true 	
+|=======================================================================
+
+Notes:
+
+* Activity Logs can have the following categories: "Administrative", "ServiceHealth", "Alert", "Autoscale", "Security"
+* SQL Diagnostics can have the following categories: "Metric", "Blocks", "Errors", "Timeouts", "QueryStoreRuntimeStatistics", "QueryStoreWaitStatistics", "DatabaseWaitStatistics", "SQLInsights"
+
+Microsoft documents Activity log schema
+https://docs.microsoft.com/en-us/azure/monitoring-and-diagnostics/monitoring-activity-log-schema[here].
+The SQL Diagnostics data is documented
+https://docs.microsoft.com/en-us/azure/sql-database/sql-database-metrics-diag-logging[here].
+Elastic does not own these data models, and as such, cannot make any
+assurances of information accuracy or passivity.
+
+===== Special note - Properties field
+
+Many of the logs contain a "properties" top level field. This is often where the
+most interesting data lives. There is not a fixed schema between log types for
+properties fields coming from different sources. This can cause mapping errors
+when shipping the data to Elasticsearch. 
+
+For example, one log may have
+properties.type where one log sets this a String type and another sets this an
+Integer type. To avoid mapping errors, the original properties field is moved to
+<azure.group>_<azure_category>_properties.<original_key>.  
+For example
+properties.type may end up as sql_diagnostics_Errors_properties.type or
+activity_log_Security_properties.type depending on the group/category from where
+the event originated.
+
+
+
+==== Testing
```

Comment:
let's leave this section out entirely ... i think it makes for the readme, but not the ascii doc.